### ISSUE TITLE
How to show Training accuracy

### ISSUE THREAD

**Issue by tiny-dentist** [mquad/hgru4rec#17]
Hi, is there any way where we can print the training accuracy percentage ?

**Comment by mquad** [comment]
Hi, the code currently logs the training and validation losses only at [this line](https://github.com/mquad/hgru4rec/blob/master/src/hgru4rec.py#L865).
You can probably extend it to compute any additional metric of your interest by calling the [evaluate_session_batch](https://github.com/mquad/hgru4rec/blob/master/src/evaluation.py#L18) function on the training data for every epoch.
